onnxruntime/test/optimizer/transpose_optimizer_test.cc

```diff
@@ -498,6 +509,7 @@
                     /*opset_version*/ 13);
 }
 
+#endif
 TEST(TransposeOptimizerTests, TestAdd) {
```

Comment:
nit: `#endif // !defined(USE_CUDA) && !defined(USE_ROCM)` would be nicer

Reply:
Thanks. I will include the change in another PR to avoid running all the CIs again.